### PR TITLE
Added env vars to config.toml to align with v5.0.x 

### PR DESCRIPTION
**Pull request by ValClarkson**
Added env vars to config.toml to align with what v5.0.x has  and updated the components.md to utilize those env vars

Issue: [sc-13842]

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [ ] Bug fix
 - [x] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Other Information**:

### CODE REVIEW

File: docs/config.toml [hunk 1]
```diff
@@ -39,6 +39,14 @@ imageCrunchyPGAdmin = "registry.developers.crunchydata.com/crunchydata/crunchy-p
...
 repositoryPrivate = "registry.developers.com/crunchydata"
 postgresOperatorTag = "ubi8-5.1.0-0"
+PGBouncerComponentTagUbi8 = "ubi8-1.16-1"
+PGBouncerComponentTagCentos8 = "centos8-1.16-1"
+PGBouncerTagUbi8 = "ubi8-5.1.0-0"
+PGBouncerTagCentos8 = "centos8-5.1.0-0"
+postgres14GIS31ComponentTagUbi8 = "ubi8-14.1-3.1-1"
+postgres14GIS31ComponentTagCentos8 = "centos8-14.1-3.1-1"
+postgres14GIS31TagUbi8 = "ubi8-14.2-3.1-5.1.0-0"
+postgres14GIS31TagCentos8 = "centos8-13.6-3.1-5.1.0-0"
```
Comment (tjmoore4) [inline docs/config.toml:49]: :wrench: This should be for PG 14 I think

File: docs/config.toml [hunk 2]
```diff
@@ -39,6 +39,14 @@ imageCrunchyPGAdmin = "registry.developers.crunchydata.com/crunchydata/crunchy-p
 repository = "registry.developers.crunchydata.com/crunchydata"
 repositoryPrivate = "registry.developers.com/crunchydata"
 postgresOperatorTag = "ubi8-5.1.0-0"
+PGBouncerComponentTagUbi8 = "ubi8-1.16-1"
+PGBouncerComponentTagCentos8 = "centos8-1.16-1"
+PGBouncerTagUbi8 = "ubi8-5.1.0-0"
+PGBouncerTagCentos8 = "centos8-5.1.0-0"
+postgres14GIS31ComponentTagUbi8 = "ubi8-14.1-3.1-1"
+postgres14GIS31ComponentTagCentos8 = "centos8-14.1-3.1-1"
+postgres14GIS31TagUbi8 = "ubi8-14.2-3.1-5.1.0-0"
```
Comment (tjmoore4) [inline docs/config.toml:48]: :question: Is `14.2` correct here or should it be `14.1`?

**Review by tjmoore4 — COMMENTED** [review]
Basic approach looks great, a couple things to review/update.

File: docs/config.toml (hunk 2 above)
Comment (ValClarkson) [reply docs/config.toml]: you are correct and just pushed an update

File: docs/config.toml (hunk 1 above)
Comment (ValClarkson) [reply docs/config.toml]: updated

**Review by tjmoore4 — APPROVED** [review]

**Pull request merged**